mageia/README.md

```diff
@@ -1,6 +1,6 @@
 # Supported tags and respective `Dockerfile` links
 
--	[`latest`, `5` (*5/Dockerfile*)](https://github.com/juanluisbaptiste/docker-brew-mageia/blob/85a4253a175cde455b9926b4a83fb8d5ceea096f/5/Dockerfile)
+-	[`latest`, `5` (*5/Dockerfile*)](https://github.com/juanluisbaptiste/docker-brew-mageia/blob/master/5/Dockerfile)
```

Comment:
This line is generated automatically from the content of `official-images/library/mageia`, so it will get updated to the newest file each time a PR is merged over there.